### PR TITLE
Added blog about Trojan (Horse) unicode characters

### PR DESCRIPTION
**Pull request by Corniel**

### CODE REVIEW

File: hello-world/2021-11-22_trojan-characters.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# (Potential) Trojan Horse (unicode) characters
+
+Most readers are familiar (enough) with Greek mythology to know about the
+origin of the Trojan Horse. 
+![Trojan Horse](2021-11-22_trojan-characters.jpg)
+
+Because of the similarities, a particular kind of [malware](https://en.wikipedia.org/wiki/Trojan_horse_(computing))
+has be named after it, and recently, a second technique to breach security
```
Comment (LauraXiulan) [inline hello-world/2021-11-22_trojan-characters.md]: 'was named after it' or 'has been named after it'

File: hello-world/2021-11-22_trojan-characters.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+measures, followed.
+
+The idea of this technique is to use (unicode) characters that not visible/rendered
```
Comment (LauraXiulan) [inline hello-world/2021-11-22_trojan-characters.md]: '...that **are** not visible/rendered...'

File: hello-world/2021-11-22_trojan-characters.md (hunk 1 above)
Comment (LFrensen) [inline hello-world/2021-11-22_trojan-characters.md]: .... has **been** named after it, ......

File: hello-world/2021-11-22_trojan-characters.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+by your IDE, so that you **think** your code works in a certain way, but the
+compiler/interpreter will behave differently (as the hacker intended).
+
+So, what kind of characters are we talking about?
+All of the following [unicode categories](https://en.wikipedia.org/wiki/Unicode_character_property) are off limits:
```
Comment (LauraXiulan) [inline hello-world/2021-11-22_trojan-characters.md:18]: What do you mean by this part?

File: hello-world/2021-11-22_trojan-characters.md (hunk 2 above)
Comment (LFrensen) [inline hello-world/2021-11-22_trojan-characters.md]: ... that **are** not visible/rendered...

**Review by LFrensen — CHANGES_REQUESTED** [review]
Suggested two changes in the file.

File: hello-world/2021-11-22_trojan-characters.md (hunk 3 above)
Comment (Corniel) [reply hello-world/2021-11-22_trojan-characters.md:18]: which unicode characters are potential trojan horses.

**Review by LFrensen — APPROVED** [review]

**Pull request merged**